src/routes/Apps/App/TrainDialogs.tsx

```diff
@@ -150,14 +150,15 @@ const getDialogKey = (trainDialog: OF.IObjectWithKey) => (trainDialog as CLM.Tra
 interface ComponentState {
     columns: IRenderableColumn[]
     sortColumn: IRenderableColumn
-    activityHistory: Activity[]
+    activityHistory: BB.Activity[]
     lastAction: CLM.ActionBase | null
     isTeachDialogModalOpen: boolean
     isEditDialogModalOpen: boolean
     isTranscriptImportOpen: boolean
     isImportWaitModalOpen: boolean
     importIndex: number | undefined
     importedTrainDialogs: CLM.TrainDialog[] | undefined
+    importedLGItems: CLM.LGItem[] | undefined
```

Comment:
Hmm, I thought you had changed preferences after seeing the `lgName` addition, but now I see this and realize it was a trick. What is the logic for capitalization?
We have mixes of `LGItem` `lgItem` and others. Whatever the pattern is, it should be clear enough that other people can repeat it.

Reply:
In my opinion, variable names shouldn't start with caps, thus "lgItems".  However if it's in the middle leave it caps.  'importedlgItems" is less readable than "importedLGItems".

Reply:
Well would be `importedLgItems` it's `importedlgItems` which avoids the i L issue, but that's only specific because of this word anyways. I don't agree it's more readable but at least I understand why they're different now.

Reply:
My other thought is that LG is an abbreviation itself, so fro me Lg reads strange